modules/@apostrophecms/notification/index.js

```diff
@@ -124,7 +125,8 @@ module.exports = {
         dismiss,
         icon,
         type,
-        buttons
+        buttons,
+        return: true
```

Comment:
We want to return the ID, why would we return an empty object 🤔

Reply:
We should skip this and always return the noteId then